jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/form/ReadonlyPlainTextTests.java

```diff
@@ -42,6 +45,12 @@ public void isValidationTest(ReadonlyPlainText item) {
         item.is().core().attr("type", "text");
     }
 
+    @Test(dataProvider = "itemsUnderTesting", expectedExceptions = {InvalidElementStateException.class})
+    public void check(ReadonlyPlainText item) {
+        item.setValue(text);
+        item.sendKeys(text);
```

Comment:
Second string will never run, because of Exception in the first string. If you want to check both methods you should split this test for two, if not - just remove second string.

Reply:
Fixed